src/Exclusion/ExpressionLanguageExclusionStrategy.php

```diff
@@ -20,7 +20,7 @@
  *
  * @author Asmir Mustafic <goetas@gmail.com>
  */
-final class ExpressionLanguageExclusionStrategy
+final class ExpressionLanguageExclusionStrategy implements ExclusionStrategyInterface
```

Comment:
I saw that this class is also a part of "exclusionStrategy", it was missing the interface implementation.